movieApp.js

```diff
@@ -1,204 +1,279 @@
+    const movieDetails = document.createElement("div");
+    movieDetails.className = "movie-details";
+    movieDetails.style.display = "none";
+    movieDetails.style.color = "white";
+    movieDetails.innerHTML = `
            <p><strong>Title:</strong> ${movie.title}</p>
            <p><strong>Description:</strong> ${movie.description}</p>
            <p><strong>Release Year:</strong> ${movie.movie_year}</p>
            <p><strong>Director:</strong> ${movie.director}</p>
            <p><strong>Actors:</strong> ${movie.actors.join(", ")}</p>
        `;
-  movieContainer.appendChild(movieDetails);
-
-  // Adding the comment Section
-
-  function createCommentSection() {
-    const commentSection = document.createElement("div");
-    commentSection.innerHTML = "Comments: ";
-    commentSection.style.fontWeight = "bold";
-    commentSection.className = "comment-section";
-    const commentInput = document.createElement("input");
-    commentInput.type = "text";
-    commentInput.value = "";
-    commentInput.placeholder = "Add a Comment";
-    commentInput.style.color = "white";
-    const commentButton = document.createElement("button");
-    commentButton.className = "comment-button";
-    commentButton.innerHTML = "Submit";
-    const comments = document.createElement("div");
-    comments.className = "comments";
-    const commentUl = document.createElement("ul");
-
-    commentButton.addEventListener("click", () => {
+    storeMovieDetail.appendChild(movieDetails);
+    movieDataContainer.appendChild(storeMovieDetail);
+
+    // Adding the comment Section
+
+    function createCommentSection() {
+      const commentSection = document.createElement("div");
+      commentSection.innerHTML = "Comments: ";
+      commentSection.style.fontWeight = "bold";
+      commentSection.className = "comment-section";
+      const commentInput = document.createElement("input");
+      commentInput.type = "text";
+      commentInput.value = "";
+      commentInput.placeholder = "Add a Comment";
+      commentInput.style.color = "white";
+      const commentButton = document.createElement("button");
+      commentButton.className = "comment-button";
+      commentButton.innerHTML = "Submit";
+      const comments = document.createElement("div");
+      comments.className = "comments";
       const commentUl = document.createElement("ul");
-      if (commentInput.value) {
-        const comment = document.createElement("li");
-        comment.textContent = commentInput.value;
-        commentUl.appendChild(comment);
-        commentInput.value = "";
-      }
+
+      commentButton.addEventListener("click", () => {
+        if (commentInput.value) {
+          const comment = document.createElement("li");
+          comment.textContent = commentInput.value;
+          commentUl.appendChild(comment);
+          commentInput.value = "";
+          // const comments = JSON.parse(localStorage.getItems("comments"));
```

Comment:
Not sure if I wrote this for you before as well, but in general, it is better not to commit commented code into the repository. We have Git and we can always go back and know what code was removed or changed 😉

Reply:
noted